src/pages/aura.f7.html

```diff
@@ -55,11 +61,14 @@
         });
       },
       resizeMessagesContent: function (){
+        let logo_aura = Dom7('#logo_aura');
         let aura = Dom7('#tab_aura');
         let messagebar = Dom7('.messagebar');
         let messagesContent = Dom7('.messages-content');
         let height = aura.height() - messagebar.height() +'px';
         messagesContent.css('height', height);
+        console.log(logo_aura.height());
+        messagesContent.css('padding-top', logo_aura.height()+10+'px');
```

Comment:
```suggestion
        messagesContent.css('padding-top', logo_aura.height() + 10 + 'px');
```
Colocar espaços entre operadores `+`, `-`, etc ajuda a legibilidade.